plugins/de.cau.cs.kieler.klighd.lsp/src/de/cau/cs/kieler/klighd/lsp/interactive/rectpack/Actions.xtend

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
I am just noticing: The interactive bundle could be split up into a main interactive bundle and then further bundles for each layout algorithm the interactive mode is implemented for. Maybe put that as a ticket for now.

Reply:
Would rather have packages than bundles, what would be the benefit of this?